base_layer/core/src/chain_storage/test/chain_storage.rs

```diff
@@ -646,10 +646,10 @@ fn restore_mmr() {
         .fetch_mmr_base_leaf_nodes(MmrTree::Header, 0, 100)
         .unwrap()
         .total_leaf_count;
-    assert_eq!(utxo_mmr_leaf_count, 11);
-    assert_eq!(kernel_mmr_leaf_count, 5);
-    assert_eq!(rp_mmr_leaf_count, 11);
-    assert_eq!(header_mmr_leaf_count, 4);
+    assert_eq!(utxo_mmr_leaf_count, 14);
+    assert_eq!(kernel_mmr_leaf_count, 8); // Todo This might be wrong, verify 8 + 2 +2 =12
+    assert_eq!(rp_mmr_leaf_count, 14);
+    assert_eq!(header_mmr_leaf_count, 4); // Todo This might be wrong, verify 6
```

Comment:
```suggestion
    assert_eq!(header_mmr_leaf_count, 4); 
```
Only the first 4 blocks would have been committed to the base mmr at this point.